opa/models/session.py

```diff
@@ -0,0 +1,65 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class Session(models.Model):
+    _name = 'session'
+    _description = 'Session from an Open Academy course'
+    _rec_name = 'title'
+
+    title = fields.Char(required=True)
+    starts_on = fields.Date(required=True)
+    ends_on = fields.Date(default=fields.Date.today)
+    duration = fields.Float(string='Duration (hours)')
+    seats = fields.Integer()
+    course_id = fields.Many2one(comodel_name='course', required=True)
+    instructor_id = fields.Many2one(comodel_name='res.partner',
+                                    domain=['|', ('instructor', '=', True),
+                                            ('category_id.parent_id.name', '=', 'Teacher')])
+    attendee_ids = fields.Many2many(string='Attendees', comodel_name='res.partner', relation='session_res_partner_rel',
```

Comment:
Break line after parenthesis

Reply:
@antonag32 this is the [black](https://github.com/psf/black) formatter suggestion:

```suggestion
    attendee_ids = fields.Many2many(
        string='Attendees',
        comodel_name='res.partner',
        relation='session_res_partner_rel',
        column1='session_id',
        column2='res_partner_id',
    )
```

Reply:
Done